### PR TITLE
Adding mass spectrometry specific variables and corresponding cv's. 

### PR DESCRIPTION
**Pull request by ZofiaRudjord**
I also updated reset_database script to create cv's because this is done now in db_init.

### CODE REVIEW

File: src/odm2_postgres_api/metadata_init/data/mass_spec/mass_spec_metadata.py [hunk 1]
```diff
@@ -15,6 +16,86 @@ def mass_spec_sampling_features() -> List[SamplingFeaturesCreate]:
...
+    mass_spec_variables_list = [{
+        "variabletypecv": "Chemistry",
+        "variablenamecv": "LC_QTOF_Raw",
+        "variabledefinition": "Raw data produced by Liquid Chmoatography coupled to Quadrupole Time of Flight "
+                              "instrument",
+        "variablecode": f'mass_spec_00',
+        "nodatavalue": -9999
+    }, {
+        "variabletypecv": "Chemistry",
+        "variablenamecv": "LC_QTOF_Peaks",
```
Comment (RmStorm) [inline src/odm2_postgres_api/metadata_init/data/mass_spec/mass_spec_metadata.py:69]: Should this be LC_QTOF_mzXML

File: src/odm2_postgres_api/metadata_init/data/mass_spec/mass_spec_metadata.py [hunk 2]
```diff
@@ -15,6 +16,86 @@ def mass_spec_sampling_features() -> List[SamplingFeaturesCreate]:
...
+        "variabledefinition": "Data produced by Liquid Chmoatography coupled to Quadrupole Time of Flight "
+                              "instrument and converted to mzXML format",
+        "variablecode": f'mass_spec_01',
+        "nodatavalue": -9999
+    }, {
+        "variabletypecv": "Chemistry",
+        "variablenamecv": "LC_QTOF_Peaks",
+        "variabledefinition": "Peaks detected by Liquid Chmoatography coupled to Quadrupole Time of Flight instrument",
+        "variablecode": f'mass_spec_1',
```
Comment (RmStorm) [inline src/odm2_postgres_api/metadata_init/data/mass_spec/mass_spec_metadata.py:78]: Can this naming scheme just progress to mass_spec_02?

**Review by RmStorm — CHANGES_REQUESTED** [review]
See comments in code.

File: src/odm2_postgres_api/metadata_init/data/mass_spec/mass_spec_metadata.py (hunk 2 above)
Comment (ZofiaRudjord) [reply src/odm2_postgres_api/metadata_init/data/mass_spec/mass_spec_metadata.py:78]: The idea was that both raw and xml are actually raw data, hence kept it at "0" level as 00 and 01.

**Review by RmStorm — APPROVED** [review]
Looks great.

**Pull request merged**